### PR TITLE
CSHARP-2853: Remove “beta” references in FLE API/reference docs

### PR DESCRIPTION
**Pull request by DmitryLukyanov**

### CODE REVIEW

File: Docs/landing/data/releases.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 current = "2.9.3"
 
 [[versions]]
-  version = "2.10.0-beta1"
+  version = "2.10.0"
```
Comment (jyemin) [inline Docs/landing/data/releases.toml]: Let's revert this until ready to do the release

File: Docs/landing/data/releases.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 current = "2.9.3"
```
Comment (DmitryLukyanov) [inline Docs/landing/data/releases.toml]: Probably this one should be 2.10.0?

**Review by jyemin — CHANGES_REQUESTED** [review]
I didn't do an exhaustive search to see if anything was missed, but everything here looks good except for the change to releases.toml.

File: Docs/landing/data/releases.toml (hunk 1 above)
Comment (DmitryLukyanov) [reply Docs/landing/data/releases.toml]: done

**Comment by DmitryLukyanov** [comment]
ready for review

File: Docs/landing/data/releases.toml (hunk 2 above)
Comment (jyemin) [reply Docs/landing/data/releases.toml]: It will be changed during the release process

**Review by jyemin — APPROVED** [review]

**Review by rstam — APPROVED** [review]
LGTM

**Review by vincentkam — APPROVED** [review]
LGTM! 😸

**Pull request closed without merging**